umu-database.csv

```diff
@@ -1046,3 +1046,4 @@ Zenless Zone Zero,egs,525aa0efd70f4399b9f64bcd2a5b38c7,umu-825cd0b5286c404289d45
 Zenless Zone Zero,none,none,umu-zenlesszonezero,,Standalone PC installer
 METAL GEAR SOLID 2: SUBSTANCE,gog,2069117974,umu-2069117974,,
 Silent Hill 3,none,none,umu-silenthill3,,
+Dirt 3,none,none,umu-321040,,
```

Comment:
Possible add a note field if you're concerned about other aliases. See https://github.com/Open-Wine-Components/umu-database/blob/main/umu-database.csv#L52 for example

Reply:
done :)